### PR TITLE
On Critical Learning Theory: the hidden curriculum

### PR DESCRIPTION
**Pull request by gallardoalba**
New training: 
On Critical Learning Theory: the hidden curriculum

### CODE REVIEW

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 1]
```diff
@@ -0,0 +1,223 @@
...
+objectives:
+- To introduce the historical origins of Critical Learning Theory.
+- Systematically present the main methods associated with Critical Learning Theory.
+contributions:
+  authorship:
+    - gallardoalba
+---
+
+> <warning-title>Recommended starting point</warning-title>
+> This training can be considered as the continuation of [Principles of learning and how they apply to training and teaching ]({% link topics/contributing/tutorials/learning-principles/tutorial.md %}). If you wish to have a panoramic view of the theoretical corpus in which the **Critical Learning Theory** is embedded, we recommend to start with the above-mentioned training.
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:25]: please add a `follow_up_training` to that tutorial and a requirements to this one like

```
requirements:
  - type: "internal"
    topic_name: contributing
    tutorials:
      - learning-principles
```

though this (and the mentioned tutorial) both feel more like they belong in the `teaching` topic (@bebatut what do you think)

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 2]
```diff
@@ -0,0 +1,223 @@
...
+{: .warning}
+
+# Introduction
+
+The **Critical Learning Theory**  is a pedagogical paradigm rooted in the broader **Critical Theory** tradition originated with the **Frankfurt School** of thought, formally known as the Institute for Social Research (*Institut für Sozialforschung*), an academic hub founded in 1923 at Goethe University in Frankfurt, Germany. 
+
+Consisting of a renowned collective of critical theorists such as Max Horkheimer, Theodor Adorno, Walter Benjamin, and Herbert Marcuse, the **Frankfurt School** developed the **Critical Theory** as a multidisciplinary approach crafted with the intent to comprehend and actively confront the oppressive dynamics ingrained within industrial societies (fig. 1).
+
+![Frankfurt School picture](../../images/critical_theory/frankfurt_school.png "Left image: Max Horkheimer and Theodor Adorno in Heidelberg in 1964 at the Max Weber Institute of Sociology. Right image: Herbert Marcuse in Newton, Massachusetts in 1955. Source: Wikipedia")
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:34]:
```suggestion
![Frankfurt School picture, on the left two old bald white men dressed in suits shake hands, right an old white man relaxes in a lawnchair holding a cigar.](images/critical_theory/frankfurt_school.png "Left image: Max Horkheimer and Theodor Adorno in Heidelberg in 1964 at the Max Weber Institute of Sociology. Right image: Herbert Marcuse in Newton, Massachusetts in 1955. Source: Wikipedia")
```

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+>
+> — Kristalina Georgieva, managing director of the International Monetary Fund.
+>
+{: .quote}
+
+## Educative systems
+
+According the **Critical Theory**, educative systems are agents of ideological control linked to the principles and processes that govern the workplace; the dominant culture in educative systems is characterised by selectively ordering and legitimising privileged forms of language, social relations, life experiences and modes of reasoning ({% cite Marcuse1964 %}).
+
+The philospher Pedro García Olivo affirms that the criterion governing Western educative systems, in line with OECD recommendations, is none other than that of favouring the adaptation of students to the requirements of the established productive and political apparatus, which requires their psychological and cultural homogenisation ({% cite Olivo2005 %}).
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:66]: perhaps write `{OECD}` here and add 

```
abbreviations:
  OECD: Organisation for Economic Co-operation and Development
```

to the yaml header.

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 4]
```diff
@@ -0,0 +1,223 @@
...
+# Critical thinking
+
+> *Questioning is an exercise in dismantling those certainties that are installed as layers of truths imposing the tyranny of the obvious.*
+>
+> — Dario Sztajnzrajbe, ¿Para qué sirve la filosofía?
+{: .quote}
+
+Before going deeper, it is inportant to accurately define **critical thinking**, since it is usually concived as a nebulose concept whose applications appears to be limitated to specific situations.
+
+The most common definition comes from the **positivist tradition** in the applied sciences; it appeals to the function of teaching students to analyse and develop reading and writing tasks from the perspective of formal logical patterns of coherence, and determining if the conclusions can be derivated from the evaluated data ({% cite Giroux1988 %}). Is it enought?
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:79]:
```suggestion
The most common definition comes from the **positivist tradition** in the applied sciences; it appeals to the function of teaching students to analyse and develop reading and writing tasks from the perspective of formal logical patterns of coherence, and determining if the conclusions can be derivated from the evaluated data ({% cite Giroux1988 %}). Is it enough?
```

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 5]
```diff
@@ -0,0 +1,223 @@
...
+
+From the assumption of this conception of critical thinking a new question arises: is it really in the interest of education system to encourage critical thinking? One element that can help to answer this question is the hidden curriculum.
+
+> <question-title>Critical thinking quiz</question-title>
+>
+> 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?** <br>
+> &nbsp; a. By avoiding critical examination of societal issues. <br>
+> &nbsp; b. By promoting conformity to existing structures. <br>
+> &nbsp; c. By fostering a reflective approach to learning and encouraging critical analysis. <br>
+> &nbsp; d. By discouraging diverse perspectives.
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:109]:
```suggestion
> 1. **How does critical thinking align with the principles of both Critical Theory and Critical Learning Theory?**
>    a. By avoiding critical examination of societal issues.
>    b. By promoting conformity to existing structures.
>    c. By fostering a reflective approach to learning and encouraging critical analysis.
>    d. By discouraging diverse perspectives.
```

please use proper markdown formatting here. We can style it as a/b/c/d with CSS which feels a bit safer.

For this page we could add a bit like

```
<style>
blockquote.question ol ol li {
  list-style: lower-alpha;
}
</style>
```

in order to achieve that.

Also it's a *lot* of bold, maybe conider not bolding the questions?

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 6]
```diff
@@ -0,0 +1,223 @@
...
+
+> <comment-title>Notes on Günther Anders</comment-title>
+>
+> Günther Anders (12 July 1902 - 17 December 1992), a German-born philosopher, journalist and critical theorist, stands as a prominent figure known for his insightful reflections on technology, existentialism, and the human condition. Although Anders did not become a formal member of the **Frankfurt School**, he is considered to have exerted an important influence on some of its members, with whom he maintained a close relationship throughout his life.
+>
+> In 1923, Gunther Anders obtained his Ph.D. in philosophy from the University of Freiburg, having studied under the esteemed philosophers Martin Heidegger and Edmund Husserl.
+>
+> ![Anders picture](../../images/critical_theory/anders.png "Günther Anders. Source: soziologie.uni-freiburg.de")
```
Comment (hexylena) [inline topics/contributing/tutorials/critical_theory/tutorial.md:199]:
```suggestion
> ![An old white man wearing glasses and a suit, stares off to the right side with a very pensive thoughtful expression.](../../images/critical_theory/anders.png "Günther Anders. Source: soziologie.uni-freiburg.de")
```

**Review by hexylena — COMMENTED** [review]
Very cool @gallardoalba!

Another thing is that the commits on this branch don't seem to be attributed to you properly. Do you want to rewrite the history to re-assign the email/user information?

File: topics/contributing/tutorials/critical_theory/tutorial.md [hunk 7]
```diff
@@ -0,0 +1,234 @@
...
+>
+> > *Just as the ideas of change and movement were the intellectual foundations of the bourgeois revolution, for the legitimisation of the collapse of the old class relations, the consolidation of that revolution in the latter part of the 19th century and in the 20th century has required a different view of the world, in line with the stabilisation of society. The idea of change had to be repressed in science as it was in society. The result has been the emphasis in modern evolutionary theories on dynamic equilibrium.*
+> >
+> {: .quote author="Richard Lewontin and Richard Levins" cite="The dialectical biologist"}
+>
+> Similarly, these authors consider that **the description of evolution as producing order is also deeply ideological**.  An alternative conception is represented by the *[disorder-function paradigm]({% link topics/transcriptomics/tutorials/differential-isoform-expression/slides.html#19})*. In this sense, evolution is neither a fact nor a theory, but a way of organising knowledge about the world.
```
[the remark this answers is not in the document]
Comment (hexylena) [reply topics/contributing/tutorials/critical_theory/tutorial.md:190]:
```suggestion
> Similarly, these authors consider that **the description of evolution as producing order is also deeply ideological**.  An alternative conception is represented by the *[disorder-function paradigm]({% link topics/transcriptomics/tutorials/differential-isoform-expression/slides.html %}#19)*. In this sense, evolution is neither a fact nor a theory, but a way of organising knowledge about the world.
```

**Comment by shiltemann** [comment]
> Last but not least, this training includes personal reflections from my teaching experiences

I think this would fit best as a "training philosophy" page, which we collect here: https://github.com/galaxyproject/training-material/tree/main/topics/teaching/tutorials/experiences. Could you move it there?

**Comment by shiltemann** [comment]
@gallardoalba also ok to leave it here, just trying to find the best place, and since you mention in several places that this is your personal experience/reflections, I thought it might fit well there.

The linting is failing because of lack of DOIs for references in the bib file, can you add those (where possible)?

**Pull request closed without merging**